nvflare/lighter/utils.py

```diff
@@ -205,3 +209,117 @@ def update_storage_locations(
     json_object = json.dumps(resources, indent=4)
     with open(target_resource, "w") as outfile:
         outfile.write(json_object)
+
+
+def shutdown_system(prod_dir: str, username: str = "admin", secure_mode: bool = False, timeout_in_sec: int = 30):
```

Comment:
Should we put these methods here in "lighter"?

"lighter" should be just for setting up the "flame" or "flare".

These methods maybe should be in "nvflare/tool" or "nvflare/utils".

In addition, I also think flare api is not fuel, as it is higher level stuff.
Other stuffs inside fuel is low level stuff used by flare.

Reply:
I also feel that this should not be in the nvflare/lighter level. It should be within the nvflare tools layer or the nvflare api layer.